tests/workflows/test_snowflake.py

```diff
@@ -0,0 +1,110 @@
+import os
+import uuid
+
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+from dask.distributed import wait
+
+snowflake = pytest.importorskip("snowflake")
```

Comment:
@j-bennet It looks like this line is causing problems and the tests are being skipped see https://github.com/coiled/benchmarks/actions/runs/4942229870/jobs/8835569338?pr=827#step:6:1068 not sure

Reply:
Yes, I don't think that the condition in `meta.yaml` worked:

[recipe/meta.yaml](https://github.com/coiled/benchmarks/pull/827/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a)

Not sure why, must have something to do with `{% set python_version = environ.get("PYTHON_VERSION", '') %}`

Reply:
Maybe you need to use `{{ python_version }}` to get the variable value in the if condition? 
notice that's what we do in  `dask =={{ dask_version }}`

Reply:
I got the jinja template right, finally. Now there are dependency errors and it can't solve for the requirements. Will try to fix.

Reply:
yes this is the snowflake connector being annoying. It looks like we moved to pandas 2.0 recently in the benchmarks, and it's complaining about that

Reply:
I saw, we want pandas == 2.0.1, but snowflake-connector-python wants it lower.

```
│     └─ snowflake-connector-python 3.0.3 would require
│        └─ pandas >1.0.0,<1.6.0 , which can be installed;
```

they have an open issue about supporting 2.0, and no visible movement on it:

https://github.com/snowflakedb/snowflake-connector-python/issues/1506

Reply:
I guess if we want to have a Snowflake workflow, we'd have to drop to pandas 1.6, which would be a bummer. Perhaps we should have a separate CI job for it.

Reply:
Having a separate CI job makes sense to me as it's the only solution, and this will keep happening with snowflake because the connector is always behind with the dependencies support. That being said. I think before doing anything further, this deserves a broader discussion, maybe at some point next week with the team? 
It'll probably be good having a summary in the main conversation of the PR

Reply:
I added a note in summary.